mizar/common/common.py

```diff
@@ -44,6 +44,12 @@ def run_cmd(cmd):
     returncode = result.returncode
     return (returncode, text)
 
+def count_characters(self, a_dict):
```

Comment:
I will be using json.dumps instead of stringify : https://stackoverflow.com/questions/46227854/json-stringify-javascript-and-json-dumps-python-not-equivalent-on-a-list
because space also count as one character.